packages/plugin-webpack/plugin.js

```diff
@@ -189,6 +184,7 @@ module.exports = function plugin(config, args) {
           entry,
           output: {
             path: destDirectory,
+            publicPath: baseUrl,
```

Comment:
needs to be set, otherwise files are always loaded relative to the current URL (breaks if the SPA owns routing).

Reply:
Ah good catch